plugins/lime-plugin-pirania/src/pages/home.js

```diff
@@ -0,0 +1,106 @@
+import { Box } from '../../../../src/components/box';
+
+export default function HomePiraniaPage ({ logged, handlePassword, submit, provider, community, member }) {
+  const now = new Date()
+  const date = now.getDate()
+  const hasPassed = date > community.payday
+  const nextPayday = hasPassed ? new Date(now.getFullYear(), now.getMonth()+1, community.payday) : now
+  const month = nextPayday.getMonth() + 1
+  const year = nextPayday.getFullYear()
+  const daysLeft = Math.floor((Date.UTC(nextPayday.getFullYear(), nextPayday.getMonth(), nextPayday.getDate()) - Date.UTC(now.getFullYear(), now.getMonth(), now.getDate()) ) /(1000 * 60 * 60 * 24))
+  const payday = community.payday === date ? 'Today' : `${community.payday}/${month}/${year}`
+  return (
+    <div>
+      <Box title={I18n.t('Users')}>
+        <div class="info">
+          <span>
+            <b>{I18n.t('Value per person')}: </b>
+            <span>{community.currency} {member.cost}</span>
+            <br />
+          </span>
+          <span>
+            <b>{I18n.t('Active member vouchers')}: </b>
+            <span>30</span>
+            <br />
+          </span>
+          <span>
+            <b>{I18n.t('Active visitor vouchers')}: </b>
+            <span>18</span>
+            <br />
+          </span>
+        </div>
+      </Box>
+      <Box title={I18n.t('Connection')}>
+        <span>
+          <b>{I18n.t('Contracted internet speed')}: </b>
+          <span>{provider.speed}</span>
+          <br />
+        </span>
+        <span>
+          <b>{I18n.t('Speed test')}: </b>
+          <span>
+            {' '}
+            {navigator.connection ? (
+              navigator.connection.downlink + 'Mbs'
+            ) : (
+              <i>{I18n.t('please use Chrome browser')}</i>
```

Comment:
Maybe it can say "speed test not supported in this browser", cause in the future the API might be supported. API support is not related with the browser, but to their advancement and their agreement/disagreement.